ledger/src/bank_forks_utils.rs

```diff
@@ -123,7 +123,7 @@ pub fn load_bank_forks(
         if process_options
             .accounts_db_config
             .as_ref()
-            .and_then(|config| config.filler_account_count)
+            .and_then(|cfg| cfg.filler_accounts.as_ref().and_then(|facfg| facfg.count))
```

Comment:
After changing the `AccountsDbConfig` to hold a `FillerAccountsConfig`, this should likely be:

```rust
.and_then(|config| config.filler_accounts_config.count)
```